src/main/java/com/buuz135/industrial/block/agriculturehusbandry/tile/PlantSowerTile.java

```diff
@@ -68,7 +68,8 @@ public WorkAction work() {
             }
             if (!stack.isEmpty() && stack.getItem() instanceof BlockItem && ((BlockItem) stack.getItem()).getBlock() instanceof IPlantable) {
                 Block block = ((BlockItem) stack.getItem()).getBlock();
-                if (block.canSustainPlant(this.world.getBlockState(pos.down()), world, pos, Direction.UP, (IPlantable) block)) {
+                Block soil = this.world.getBlockState(pos.down()).getBlock();
+                if (soil.canSustainPlant(this.world.getBlockState(pos.down()), world, pos, Direction.UP, (IPlantable) block)) {
```

Comment:
Wouldn't it be better/cleaner to do this as:
```java
if (this.world.getBlockState(pos.down()).canSustainPlant(world, pos, Direction.UP, (IPlantable) block)) {
```
Even though I think it really should be:
```java
if (this.world.getBlockState(pos.down()).canSustainPlant(world, pos.down(), Direction.UP, (IPlantable) block)) {
```

Reply:
I tried with custom farmland and it worked. If Buuz don’t want the variable, he can just do it on his own and close this PR. It’s not very much code :)

Reply:
The main thing with the secondary line, that I think is correct, is also to make sure that it fixes it to be giving the custom farmland the  proper position it is at.

Reply:
Oh, with mobile phone it commented at the wrong place. I tried both methods and both are working.. I'll push the second version. Buuz can just close this PR because I commited too much :D it's just for him to see the final code.